itests/standalone/smoke/pom.xml

```diff
@@ -39,12 +39,27 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-spring-javaconfig</artifactId>
```

Comment:
Why do we need these new dependencies?

Reply:
They were needed two months ago, but now when re-checking they are not needed anymore. Apparently the WF Galleon pack brings them now.